config/coordinating-skeleton-updates.md

```diff
@@ -0,0 +1,34 @@
+# Coordinating Skeleton Updates #
```

Comment:
```suggestion
# Coordinating skeleton updates #
```

Reply:
I believe our style docs say that first-level headings are title case, while subheadings are sentence case.  I think @hillaryj is correct here.

Reply:
Arg- my bad.  I was looking elsewhere in the doc and got it wrong.  Sorry!

Reply:
First-level headings are title-cased, per our style guide.